app/src/main/java/com/codepath/chefster/activities/MainActivity.java

```diff
@@ -83,6 +83,7 @@ protected void onCreate(Bundle savedInstanceState) {
         loadDishes();
 
         setViewPager();
+        loadDataToDatabase();
```

Comment:
After using the loadDataToDatabase we need to comment it out, 
because changes made on the database will run over, 
we should think to sync the database into the .json modified it and then load it back.